Features/ManageabilityPkg/Universal/IpmiSmbiosTransferDxe/IpmiSmbiosTransferDxe.inf

```diff
@@ -0,0 +1,57 @@
+## @file
+#  A simple implementation to transfer SMBIOS tables to a BMC
+
+#  SPDX-FileCopyrightText: copyright (c) 2022-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = IpmiSmbiosTransfer
+  FILE_GUID                      = 1e2f6b56-4675-446b-9c0a-1eb66e50c840
+  MODULE_TYPE                    = DXE_DRIVER
+  VERSION_STRING                 = 1.0
+  ENTRY_POINT                    = IpmiSmbiosTransferEntry
+
+#
+#  VALID_ARCHITECTURES           = IA32 X64 EBC
+#
+
+[Sources]
+  IpmiSmbiosTransferDxe.c
+
+[LibraryClasses]
+  BaseMemoryLib
+  DebugLib
+  UefiBootServicesTableLib
+  UefiRuntimeServicesTableLib
+  UefiDriverEntryPoint
+  UefiLib
+  MemoryAllocationLib
+  ReportStatusCodeLib
+  BaseCryptLib
+
+[Packages]
+  MdePkg/MdePkg.dec
+  MdeModulePkg/MdeModulePkg.dec
+  CryptoPkg/CryptoPkg.dec
+  DynamicTablesPkg/DynamicTablesPkg.dec

```

Comment:
Ah, can you elaborate further why it is relevant to DynamicTablesPkg?

Reply:
No it is not. I removed DynamicTablesPkg.